CHANGES.txt

```diff
@@ -1,3 +1,12 @@
+2020-01-31 version 3.11.3 (C++/Java/Python/PHP/Objective-C/C#/Ruby/JavaScript)
+
+  C++
+  * Add OUT and OPTIONAL to windows portability files (#7087)
+
+  PHP
+  * Refactored ulong to zend_ulong for php7.4 compatibility (#7147)
```

Comment:
There is a change to fix segment fault in php.

Reply:
Updated.